src/passes.jl

```diff
@@ -306,6 +306,7 @@ end
 function process!(::SetupBuildDirectory, page::Page, content, block)
     dir = dirname(page.build)
     isdir(dir) || mkpath(dir)
+    addstaticfiles!(dir)
```

Comment:
I think `SetUpBuildDirectory` would run on every `page`, so if there's any `page` in a subdirectory then this will add the static files to that dir as well, i.e.

```
build/
    Lapidary.css
    manual/
        Lapidary.css
    ...
```

Is that then intended behaviour?

Would just need to check that the page's parent directory is the build dir itself to fix this I believe.